src/AnalyzerConfiguration/NI.CSharp.Analyzers.targets

```diff
@@ -26,6 +26,22 @@
     <NI1704_AdditionalSpellingDictionary>$(NI_CodeAnalysisRuleSetDirectory)\NI1704_AdditionalSpellingDictionary.dic</NI1704_AdditionalSpellingDictionary>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(NI_EnableAnalyzers)' == 'True' and '$(AnalysisMode)' == 'NI'">
+    <AnalysisModeStyle Condition="'$(AnalysisModeStyle)' == ''">Default</AnalysisModeStyle>
```

Comment:
Why is this category and the one below set to 'Default' vs 'All'?

Reply:
When enabling all security rules in ASW, there are a ton of violations around dll imports not specifying a default search path.  I had tried to add a default search path but tests started to fail.  I did not have time to go through resolving it, so I went with the default rules for now.

The style rules are set to default because they are very noisy and we don't want them all enabled.  It was simpler to use default and then disable certain rules that were still too noisy.